src/Uno.UI.Toolkit/ElevatedView.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Analyzer 'SonarAnalyzer.Rules.CSharp.ObjectsShouldNotBeDisposedMoreThanOnce' threw an exception of type 'SonarAnalyzer.SymbolicExecution.SymbolicExecutionException' with message 'Error processing method: UpdateElevation ## Method file: ElevatedView.cs ## Method line: 112,2 ## Inner exception: System.NullReferenceException: Object reference not set to an instance of an object ##   at SonarAnalyzer.SymbolicExecution.CSharpExplodedGraph.VisitDefaultExpression (Microsoft.CodeAnalysis.SyntaxNode instruction, Son](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5093276)